zkevm-circuits/src/evm_circuit/util/math_gadget.rs

```diff
@@ -533,33 +536,36 @@ impl<F: FieldExt, const NUM_BYTES: usize> RangeCheckGadget<F, NUM_BYTES> {
 }
 
 /// Returns `1` when `lhs < rhs`, and returns `0` otherwise.
-/// lhs and rhs `< 256**NUM_BYTES`
-/// `NUM_BYTES` is required to be `<= MAX_BYTES_FIELD` to prevent overflow:
-/// values are stored in a single field element and two of these
-/// are added together.
+/// lhs and rhs `< 256**N_BYTES`
+/// `N_BYTES` is required to be `<= MAX_N_BYTES_INTEGER` to prevent overflow:
```

Comment:
I'm confused here.
`256**N_BYTES`  would be the equivalent to `32**(N_BYTES *8)`. But not entirely sure what this value represents TBH.

Reply:
Ohhh you do it for the range over 256. Is that the case? If so, now all makes sense..

Reply:
Yes, it verifies the range by doing `N` times `Range256` range lookup. We can further optimize it with larger chunk if needed.